### DISCUSSION TITLE
Question about Layout Settings

### DISCUSSION THREAD

**Discussion by jam1974** [templaza/astroid-framework#125] [Q&A] [answered]
When I set the Layout Settings to Boxed, I get a Border or Shadow on the web page. How can I disable the Border or Shadow?

**Comment by sonvnn** [comment]
You can use custom css feature

```
.astroid-layout.astroid-layout-boxed .astroid-wrapper {
    box-shadow: none;
}
```

**Reply by jam1974** [reply]
Unfortunately, a frame remains around the header section.

**Comment by jam1974** [comment, marked as answer]
This works for me. If it is the best solution, I don't know. 

.astroid-layout.astroid-layout-boxed .astroid-wrapper {
    box-shadow: none !important;
}

.border-bottom, .astroid-header-section {
  border-bottom: none !important;
  box-shadow: none !important;
}

**Reply by Chacapamac** [reply]
@jam1974 Yep, that seem right, no choice to add that if you want a clean look
